apostrophecms/apostrophe
# hide save button if the in-context document is autopublished

## Summary

Hide save button if in-context document is autopublished

## What are the specific steps to test this change?

1. Set a piece or page option to `autopublish: true`
2. Open the preview of a document of that type
3. The save button should not be available in-content, same for the toggle `draft/published` next to the title
4. Do the same with templates from `@apostrophecms/doc-template-library`

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
